languages/zig.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+set -e
+
+# Usage:   ./languages/zig.sh INPUT                 OUTPUT                 SOLUTION
+# Example: ./languages/zig.sh days/day-03/input.txt days/day-03/output.txt days/day-03/solutions/main.zig
+
+INPUT="$1"
+OUTPUT="$2"
+SOLUTION="$3"
+
+cat $INPUT | zig run $SOLUTION -O ReleaseFast | diff - $OUTPUT
```

Comment:
Bonus points for `zig run` instead of `zig compile && ./run` :+1: Looks good :smiley: